renku/cli/init.py

```diff
@@ -443,6 +446,14 @@ def init(
         except GitCommandError as e:
             click.UsageError(e)
 
+    # supply additional parameters
+    parameter["__template_source__"] = template_source
+    parameter["__template_ref__"] = template_ref
+    parameter["__template_id__"] = template_id
+    parameter["__namespace__"] = ""
+    parameter["__repository__"] = ""
+    parameter["__project_slug__"] = ""
+
```

Comment:
`project_slug` includes the namespace - there are places where we want to use the escaped and stripped project name for naming certain files. Could we add `project_name_stripped` (from [here](https://github.com/SwissDataScienceCenter/renku-python/pull/1469/files#diff-35c86f980dbff80762ed1851377d73e2R71) but maybe give it a nicer name?) to the parameters that are passed to the template?